### PR TITLE
Release 3.68.0

### PR DESCRIPTION
**Pull request by jstvz**
(Make changes to the Markdown version)

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,27 @@
 History
 =======
 
+3.68.0 (2022-10-31)
+-------------------
+
+Critical Changes 🎉
+
+-  Upgraded RobotFramework from version 4.1.3 to version 6. As with most robotframework updates, there are a host of new features and a few deprecated features and backwards incompatibilities. For a complete list consult the `robotframework release notes <https://github.com/robotframework/robotframework/releases>`__ in https://github.com/SFDO-Tooling/CumulusCI/pull/3417
```
Comment (davidmreed) [inline HISTORY.rst:10]:
```suggestion
-  Upgraded Robot Framework from version 4.1.3 to version 6. As with most Robot Framework updates, there are a host of new features and a few deprecated features and backwards incompatibilities. For a complete list consult the `Robot Framework release notes <https://github.com/robotframework/robotframework/releases>`__ in https://github.com/SFDO-Tooling/CumulusCI/pull/3417
```

File: docs/history.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
 # History
 
+## 3.68.0 (2022-10-31)
+
+Critical Changes 🎉
+
+-   Upgraded RobotFramework from version 4.1.3 to version 6. As with most robotframework updates, there are a host of new features and a few deprecated features and backwards incompatibilities. For a complete list consult the [robotframework release notes](https://github.com/robotframework/robotframework/releases) in https://github.com/SFDO-Tooling/CumulusCI/pull/3417
```
Comment (davidmreed) [inline docs/history.md:7]:
```suggestion
-   Upgraded Robot Framework from version 4.1.3 to version 6. As with most Robot Framework updates, there are a host of new features and a few deprecated features and backwards incompatibilities. For a complete list consult the [Robot Framework release notes](https://github.com/robotframework/robotframework/releases) in https://github.com/SFDO-Tooling/CumulusCI/pull/3417
```

**Review by davidmreed — APPROVED** [review]

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,27 @@
...
+
+Changes 🎉
+
+-  Add support for InstallKey, sourceOrg, release keys on 2GP version create by @davidmreed in https://github.com/SFDO-Tooling/CumulusCI/pull/3403
+-  We now support the ``Territory2``, ``Territory2Model``, ``Territory2Type``, and ``Territory2Rule`` MetaData types. by @TheBitShepherd in https://github.com/SFDO-Tooling/CumulusCI/pull/3406
+-  Add ``push_upgrade_org`` flow to test in simulated push upgrade environment by @davidmreed in https://github.com/SFDO-Tooling/CumulusCI/pull/3286
+-  Fixed bug preventing the ``deploy_omni_studio_site_settings`` task from deploying to scratch orgs with the new ``.scratch.`` domain names by @jofsky in https://github.com/SFDO-Tooling/CumulusCI/pull/3411
+-  All new keychain details are serialized using JSON, not pickle by @bethbrains in https://github.com/SFDO-Tooling/CumulusCI/pull/3390
+
```
Comment (prescod) [inline HISTORY.rst:19]:
```suggestion
- Snowfakery (upgraded to 3.4) can now generate Event or Meeting Schedules similar to Calendar Apps
- Snowfakery now has a Salesforce.ContentFile feature for generating Content Versions

```

**Comment by prescod** [comment]
Can we please standardize on a Release Notes format and stick with it? Some releases we give credit. Some we don't. Some releases (this one in particular) we give URLs to pull requests. Some we don't.

**Review by prescod — CHANGES_REQUESTED** [review]
Please see my two comments above before merging.

**Review by prescod — APPROVED** [review]

**Pull request merged**